pkg/build/builder/sti.go

```diff
@@ -384,3 +393,13 @@ func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
 	}
 	return config, nil
 }
+
+func copyArtifactSourceList(artifactsMapping []api.ImageSourcePath) (volumeList []s2iapi.VolumeSpec) {
```

Comment:
1. good point
2. that is your preference - i'd have to add another line where i declare the variable 
my preference is to code as little as i can. however -  i do agree, it shouldn't be used in short functions just like this one.
https://tour.golang.org/basics/7